sceptre/cli/list.py

```diff
@@ -89,6 +89,21 @@ def list_outputs(ctx, path, export):
                         ),
                         "text",
                     )
+
+    elif export == "stackoutput":
+        stack_names = {stack.name: stack.external_name for stack in plan.graph}
+        for response in responses:
+            for stack_name, stack in response.items():
+                for output in stack:
+                    write(
+                        "!stack_output_external {0}::{1} [{2}]".format(
```

Comment:
These are not stack-outputs. They're external stack-outputs. I think this particular functionality seems pretty calibrated to your own particular use-case. But I would argue stack_output_external is less commonly used than `stack_output`. If you're outputting "stack outputs", I wouldn't expect them to be external ones.

Reply:
agree, have a look now.

Reply:
We use `!stack_output_external` more than `!stack_outputs`.  I actually didn't even know about the export flag until now, but this seems pretty useful for my use case.